src/Markdig.Wpf/MarkdownViewer.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+
+namespace Markdig.Wpf
+{
+    /// <summary>
+    /// A markdown viewer control.
+    /// </summary>
+    public class MarkdownViewer : Control
```

Comment:
I'm wondering if it could be a good idea to add a `Pipeline` property so that user can customize the markdown pipeline (with their own choice of extensions).
Like the previous PR we can review that later on a branch where we will cleanup the code to work nicely with extensions.